api/controllers/addRoleToUser.js

```diff
@@ -0,0 +1,10 @@
+const requireAuthentication = require('../middleware/requireAuthentication');
+
+module.exports = function(store) {
+	return [requireAuthentication(), async function(req, res) {
+		const id = req.params.id;
+		if (!req.identity.is(`admin@users/${id}`)) return res.failure('You do not have permissions to modify this object.', 403);
+		const statusCode = await store.addRole(id, req.body.role, req.body.scope);
+		return res.success(null, statusCode);
```

Comment:
See the other code. Return the new array of roles instead, and let's keep the code at `200`.